includes/course-theme/class-sensei-course-theme.php

```diff
@@ -49,79 +56,146 @@ public static function instance() {
 
 	/**
 	 * Initializes the Course Theme.
-	 *
-	 * @param Sensei_Main $sensei Sensei object.
 	 */
-	public function init( $sensei ) {
-		add_action( 'admin_enqueue_scripts', [ $this, 'add_feature_flag_inline_script' ] );
-
-		if ( ! $sensei->feature_flags->is_enabled( 'course_theme' ) ) {
-			// As soon this feature flag check is removed, the `$sensei` argument can also be removed.
-			return;
-		}
+	public function init() {
+		add_action( 'setup_theme', [ $this, 'add_rewrite_rules' ], 0, 10 );
```

Comment:
Just to avoid introducing unwanted features before launching the project, I think we can add a feature flag check here too, or we can just move the check from the other class to the `includes/class-sensei.php`.

Reply:
Ah good catch, adding it in https://github.com/Automattic/sensei/pull/4479/commits/5337713507c97adb961f24ddc8cb54927a7dbb29